src/test/java/uk/gov/mca/serializer/BeaconsSearchResultSerializerTest.java

```diff
@@ -0,0 +1,136 @@
+package uk.gov.mca.serializer;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.mca.beacons.service.model.Activity;
+import uk.gov.mca.beacons.service.model.Beacon;
+import uk.gov.mca.beacons.service.model.BeaconPerson;
+import uk.gov.mca.beacons.service.model.BeaconUse;
+import uk.gov.mca.beacons.service.model.BeaconsSearchResult;
+import uk.gov.mca.beacons.service.model.Environment;
+import uk.gov.mca.beacons.service.model.Purpose;
+import uk.gov.mca.beacons.service.serializer.BeaconsSearchResultSerializer;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
Can remove this extension as it's only required when you want Mockito to auto-inject the mocks to the call under test